logic/atrium-logic-jvm/src/test/kotlin/ch/tutteli/atrium/logic/impl/creating/filesystem/hints/SymbolicLinkResolvingSpec.kt

```diff
@@ -1,3 +1,6 @@
+//TODO remove file with 1.0.0
```

Comment:
Why do you suggest to remove the file with 1.0.0?

Reply:
that’s a copy & paste error.